README.rst

```diff
@@ -779,6 +779,76 @@ values of this attribute, ordered in priority from high to low. Using this optio
 label. In this example an RQT item with multiple TEST items, one with a *fail* and others a *pass* as *result* value in
 the TEST_REP item, will be added to the *fail* slice of the pie chart.
 
+
+Retrieving checklist values from GitLab/GitHub (advanced)
+=========================================================
+
+The plugin can add an additional attribute to a traceability item if its item ID exists in a checklist inside the
+description of a merge/pull request. Documentation items can be linked to a checklist by defining them with the
+*checklist-item* directive.
+
+.. code-block:: rest
+
+    .. checklist-item:: PLAN-UNIT_TESTS Have you added unit tests for regression detection?
+
+This directive gets treated the same as the *item* directive, except that an additional attribute gets added when the
+item's ID exists in the checklist. The attribute's value depends on the status of the checkbox. The plugin queries the
+API of GitLab/GitHub for the description of the configured merge request.
+
+The configuration of this feature is stored in the configuration variable *traceability_checklist*.
+
+.. code-block:: python
+
+    traceability_checklist = {
+        'private_token': config('PRIVATE_TOKEN'),
+        'api_host_name': config('API_HOST_NAME'),
+        'project_id': config('PROJECT_ID'),
+        'merge_request_id': config('MERGE_REQUEST_ID'),
+        'attribute_name': config('TARGET_ATTRIBUTE_NAME'),
+        'attribute_to_str': config('TARGET_ATTRIBUTE_TO_STRING'),
+        'attribute_values': config('TARGET_ATTRIBUTE_VALUES'),
+    }
+
+In this default configuration the variables are fetched from the environment, e.g. by using a ``.env`` file.
+
+.. code-block:: bash
+
+    # copy example .env to your .env
+    cp doc/.env.example .env
+
+    # add env variables by adjusting the template values in .env
+
+- *ATTRIBUTE_NAME* is the identifier of the attribute to be added, e.g. *checked*.
+- *ATTRIBUTE_TO_STRING* is the string representation (as to be rendered in html) of the attribute name, e.g. *Answer*.
+- *ATTRIBUTE_VALUES* are two comma-separated attribute values, e.g. *yes/no*. The first value is used when the checkbox is checked and the second value when unchecked.
```

Comment:
if they are comma serparated then use `yes/no` as `yes,no` to avoid confusion

Reply:
68fab7125c5a129aeccf9e44d45eb21a64726e2d